kidkode/src/main/webapp/script.js

```diff
@@ -1,6 +1,13 @@
 ///////////////
 // FUNCTIONS //
 ///////////////
+/*
+ * Redirect the user back to the homepage.
+ */
+function goHome() {
+    window.location.replace('/');
```

Comment:
I think this should be window.location.assign('/'); so that the user can press the back button on their browser to get back to the quiz https://stackoverflow.com/questions/4505798/difference-between-window-location-assign-and-window-location-replace